### PR TITLE
Replace Graph Loader with withMessageWrapper HOC

### PR DESCRIPTION
**Pull request by erikasv**
Created withMessageWrapper HOC to wrap all charts and use the chart directly from the components.

Some changes had to be made on the types of props and states of some components

### CODE REVIEW

File: src/pages/search/drawer/landscape/connectivity/CurrentPAConnectivity.tsx [hunk 1]
```diff
@@ -250,12 +251,11 @@ class CurrentPAConnectivity extends React.Component<Props, currentPAConnState> {
             Haz clic en un área protegida para visualizarla
           </h3>
           <div>
-            <GraphLoader
-              graphType="MultiSmallSingleBarGraph"
+            <MultiSmallSingleBarGraph
```
Comment (cazapatamar) [inline src/pages/search/drawer/landscape/connectivity/CurrentPAConnectivity.tsx:254]: There is a problem with height of this graph

imagen

File: src/pages/search/drawer/landscape/connectivity/TimelinePAConnectivity.tsx [hunk 2]
```diff
@@ -143,7 +143,7 @@ class TimelinePAConnectivity extends React.Component<
         <div>
           <h6>Conectividad áreas protegidas en el tiempo</h6>
           <div>
-            <GraphLoader
+            <MultiLinesGraph
               graphType="MultiLinesGraph"
```
Comment (cazapatamar) [inline src/pages/search/drawer/landscape/connectivity/TimelinePAConnectivity.tsx:147]: I think we should delete this prop as it is not used anymore.

Also, there is a warning related to onClickGraphHandler marked as required

imagen

**Review by cazapatamar — CHANGES_REQUESTED** [review]

**Comment by cazapatamar** [comment]
Clic is not working on SingleBulletGraph (Number of especies and Functional Diversity)

imagen

imagen

File: src/pages/search/drawer/landscape/connectivity/CurrentPAConnectivity.tsx (hunk 1 above)
Comment (erikasv) [reply src/pages/search/drawer/landscape/connectivity/CurrentPAConnectivity.tsx:254]: the worst part is that I already have seen that... don't know what happened with that change xD

File: src/pages/search/drawer/landscape/connectivity/TimelinePAConnectivity.tsx (hunk 2 above)
Comment (erikasv) [reply src/pages/search/drawer/landscape/connectivity/TimelinePAConnectivity.tsx:147]: fixed

File: src/pages/search/drawer/landscape/connectivity/CurrentPAConnectivity.tsx (hunk 1 above)
Comment (cazapatamar) [reply src/pages/search/drawer/landscape/connectivity/CurrentPAConnectivity.tsx:254]: That's normal in a huge refactor like this :D

**Comment by erikasv** [comment]
> Clic is not working on SingleBulletGraph (Number of especies and Functional Diversity)

Fixed... This wouldn't have happened with ts xD

**Review by Trjegul84 — APPROVED** [review]

**Pull request merged**